app/models/survey.rb

```diff
@@ -22,6 +23,10 @@ def is_sendable?
     first_question.present?
   end
 
+  def sent?
```

Comment:
`was_sent` or `has_been_sent` could be a good candidates for this method name. I'll leave it up to you.